src/main/java/com/microsoft/aad/msal4j/ClaimsRequest.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.Getter;
+import lombok.Setter;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the claims request parameter as an object
+ *
+ * @see <a href="https://openid.net/specs/openid-connect-core-1_0-final.html#ClaimsParameter">https://openid.net/specs/openid-connect-core-1_0-final.html#ClaimsParameter</a>
+ */
+public class ClaimsRequest {
```

Comment:
Did not we decide to go with just claims for accessToken ?

Reply:
I believe we decided to just do ID tokens. The OIDC spec seems to support custom claims in for the id_token and userinfo sections, however we talked about how the library doesn't really do anything with userinfo.

However, claims in the access_token section are needed for client capabilities, so I decided to keep the userinfo and access_token methods but make them non-public, so that our library can still use them for setting up whatever we may need to (like client capabilities), and only the id_token claims are public and configurable by users.

Reply:
As discussed, this class lets users and the library set claims in the id_token field, only the library can set claims in the accessToken field, and I removed the ability to set claims in userinfo to avoid confusion (as these claims would not really ever be used).